data/locale/German.json

```diff
@@ -11,30 +11,30 @@
 
     "Games": "Spiele",
     "All": "Alles",
-    "Press \uE000 to select": "Drücke \uE000 zum auswählen",
-    "\uE001 to return": "\uE001 für zurück",
-    "\uE002 to enter a title ID": "\uE002 zum eingeben der TID",
-    "\uE003 to search": "\uE003 zum suchen",
-    "\uE046 to open the queue": "\uE046 um die Warteschlange zu öffnen",
+    "Press \uE000 to select": "Drücke \uE000 zum Auswählen",
+    "\uE001 to return": "\uE001 , um ins vorherige Menü zu gelangen",
+    "\uE002 to enter a title ID": "\uE002 zum Eingeben einer TID",
+    "\uE003 to search": "Drücke \uE003 zum Suchen",
+    "\uE046 to open the queue": "\uE046 , um die Warteschlange zu öffnen",
 
     "Select a folder:": "Wähle einen Ordner:",
-    "\uE002 to switch to SD": "\uE002 um zu SD zu wechseln",
-    "\uE002 to switch to NAND": "\uE002 um zu NAND zu wechseln",
-    "\uE002 to switch to USB": "\uE002 um zu USB zu wechseln",
+    "\uE002 to switch to SD": "\uE002 , um zur SD-Karte zu wechseln",
+    "\uE002 to switch to NAND": "\uE002 , um zum NAND zu wechseln",
+    "\uE002 to switch to USB": "\uE002 , um zum USB-Stick zu wechseln",
     "Searching on": "Suche in",
 
-    "Press \uE000 to install to USB": "Drücke \uE000 um zu USB zu installieren",
-    "Press \uE002 to install to NAND": "Drücke \uE002 um zu NAND zu installieren",
-    "Press \uE07B to delete files after the installation": "Drücke \uE07B um die Dateien nach der Installation zu löschen",
-    "Press \uE07B to keep files after the installation": "Drücke \uE07B um die Dateien nach der Installation zu behalten",
+    "Press \uE000 to install to USB": "Drücke \uE000 , um nach USB zu installieren",
+    "Press \uE002 to install to NAND": "Drücke \uE002 , um nach NAND zu installieren",
+    "Press \uE07B to delete files after the installation": "Drücke \uE07B , um die Dateien nach der Installation zu löschen",
+    "Press \uE07B to keep files after the installation": "Drücke \uE07B , um die Dateien nach der Installation zu behalten",
 
-    "Press \uE000 to delete": "Drücke \uE000 um zu deinstallieren",
+    "Press \uE000 to delete": "Drücke \uE000 zum Deinstallieren",
 
-    "Press \uE000 to continue": "Drücke \uE000 um fortzufahren",
+    "Press \uE000 to continue": "Drücke \uE000 zum Fortsetzen",
     "Press \uE001 to return": "Drücke \uE001 um zurück zu gehen",
     "Generating fake ticket...": "Generiere fake Ticket...",
     "Fake ticket generated on:": "Fake ticket erstellt:",
-    "Press any key to return": "Drücke einen beliebigen Button um zurück zu gehen",
+    "Press any key to return": "Drücke eine beliebige Taste zur Rückkehr",
```

Comment:
This sounds weird + 3 lines above it is "um zurück zu gehen", too. Please keep both in sync